README.md

```diff
@@ -63,6 +63,8 @@ Documentation of the LSST Science Pipelines is at https://pipelines.lsst.io
         $ ingestImagesDecam.py /path/to/repo --filetype raw /path/to/raw/*.fits.fz
         $ ingestCalibs.py /path/to/repo/  --calib /path/to/calib/repo/ --calibType defect /path/to/calib/*fits --validity 999
```

Comment:
I will clarify that this suggestion is for ingesting biases and flats only, and not currently supported for fringes and defects. I will also add more examples here for how to ingest each kind of calibration product.